blog/_posts/2017-10-26-nieuwe-website.markdown

```diff
@@ -0,0 +1,55 @@
+
+* **Volgende wedstrijd**: Op de homepagina zal de volgende wedstrijd voor één van onze ploegen getoond worden.
+* **Alle wedstrijden deze week**: Deze kalender weergave toont alle wedstrijden van deze week.
+* **Per team**: Op de team detail pagina wordt automatisch de volgende wedstrijd weergegeven, verder kan je ook alle toekomstige wedstrijden en reeds gespeeld wedstrijden bekijken.
+* **Wedstrijd detail**: Via al de bovenstaande weergaves kan je doorklikken naar het wedstrijd detail. Daarop staat al de nodige info over de wedstrijd, inclusief een kaart weergave waar de wedstrijd plaatsvind (zo moet je dat niet meer zelf opzoeken), alsook de informatie die de table officials moeten opnemen in de feuille.
+
+### News & blog
+
+Deze twee secties zijn bedoeld om informatie te delen met alle leden en sympathisanten van de club. **News** omvat alle aankondigingen van activiteiten alsook belangrijke boodschappen. **Blog** omvat meer uitgebreide verhalen over de belevingen binnen de club, zoals bvb de basketba(bel)l, wedstrijd verslagen en dergelijke.
+
+In principe kan iedereen, dus ook u, nieuws items en verhalen indienen om opgenomen te worden in deze secties. De stappen om dit te verwezenlijken worden uitgelegd op [onze technische wiki](https://github.com/yvesgoeleven/BasketLummen/wiki) (uitleg in het engels).
+
+Er staat een team, de cel communicatie, klaar om elk ingediend item te reviewen en op te nemen in de website na goedkeuring.
+
+### Business sectie
+
+Er werd een volledig nieuwe **business sectie** toegevoegd. Deze heeft vooral tot doel om aan potentiele sponsors uit te leggen wat er allemaal gedaan wordt qua visibiliteit, evenementen en dergelijke.
+
+## Tekortkomingen
+
+Is alles dan perfect aan de nieuwe website? Nee, zeker niet. Er zijn een aantal tekortkomingen waar we ons bewust van zijn (en waarschijnlijk ook een aantal waar we ons niet van bewust zijn).
+
+Een van de grootste beperkingen waar we momenteel nog mee zitten is de informatie die basketbal.vlaanderen ons aanbiedt. Deze omvat enkel informatie over de ploegen die in de reguliere competitie zitten en is incompleet wat betreft de jongste jeugdploegen. Vandaar dus dat er geen spelers geregistreerd staan voor de U10 en jonger. Dit zal op termijn opgelost worden, geen nood, maar het vraagt wat tijd.
+
+Merk je verder nog iets, of heb je een suggestie om de informatie op onze website te verbeteren, dan kan je altijd een email sturen naar [yves@goeleven.com](mailto://yves@goeleven.com). Indien je meer technisch aangelegd bent, kan je ook gebruik maken van onze issue tracker op [github](https://github.com/yvesgoeleven/BasketLummen/issues).
+
+## Vorige website
+
+De vorige website, alsook de coach corner, zijn zeker nog niet verdwenen. Indien je in de nieuwe website informatie mist dan kan je de vorige nog steeds bereiken [via deze link](/b/). 
```

Comment:
ik zou het stukje rond coach corner weglaten en naar de coaches mailen. Spelers, supporters,... weten niet waar dit over gaat. Ook zou ik er een dankjewel aan Johnny inzetten. Voor de rest ziet het er prima uit!

Reply:
Lijkt me idd ook beter om dat weg te laten, roept alleen maar vragen op. Goed idee om Johnny te bedanken!

Reply:
Inderdaad, johny heeft er enorm veel tijd in gestoken, meeste mensen beseffen dat niet hoeveel tijd het vraagt over de jaren heen om zo'n systeem te onderhouden (zeker ook omdat de meesten niet weten dat er zo'n groot stuk coach corner achter zit)